docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,551 @@
+                onClick={() => {}}
+                role="button"
+              />
+            </Box>
+          ) : (
+            <AvatarGroup
+              accessibilityLabel={accessibilityLabel}
+              size="md"
+              collaborators={collaborators}
+            />
+          );
+        }}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      columns={2}
+      title="Role"
+      description="AvatarGroup can be display only but it can also act as a button or link. AvatarGroup can only be clickable if role is set to `button` or `link`. For `button` role, `onClick` is required. For `link` role, `href` is required."
+    >
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const [selectedBoard, setSelectedBoard] = React.useState('Fashion');
+  const anchorRef = React.useRef();
+
+  const SearchCollaboratorsField = () => {
+    const ref = React.useRef();
+
+    React.useEffect(() => {
+      ref.current.focus();
+    }, []);
+
+    return (
+      <SearchField
+        accessibilityLabel="Search other users by name or email"
+        id="searchField"
+        onChange={() => {}}
+        placeholder="Search by name or email"
+        size="lg"
+        ref={ref}
+      />
+    )
+  }
+
+  return (
+    <React.Fragment>
+      <Box>
+        <AvatarGroup
+          accessibilityLabel="Group collaborators: Keerthi, Alberto, and Shanice. Add collaborators to this board."
+          accessibilityExpanded={open}
+          addCollaborators
+          role="button"
+          onClick={() => setOpen(true)}
+          ref={anchorRef}
+          size="md"
+          collaborators={[
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+              {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+          ]}
+          />
+        </Box>
+      {open && (
+        <Layer>
+          <Popover
+            anchor={anchorRef.current}
+            idealDirection="down"
+            onDismiss={() => setOpen(false)}
+            positionRelativeToAnchor={false}
+            size="xl"
+          >
+            <Box width={360}>
+              <Box flex="grow" marginEnd={4} marginStart={4} marginTop={6} marginBottom={8}>
+                <Flex direction="column" gap={6}>
+                  <Text align="center" color="darkGray" weight="bold">
+                    Invite collaborators
+                  </Text>
+                  <SearchCollaboratorsField />
+                </Flex>
+              </Box>
+            </Box>
+          </Popover>
+        </Layer>
+      )}
+    </React.Fragment>
+  );
+}
+  `}
+      />
+      <MainSection.Card
```

Comment:
I'm a bit confused by this example. Is it supposed to open the popover? Or navigate away?

Reply:
Damn, I reused the example and didn't think twice. Thanks for catching it.

Reply:
DONE